pytest_fixtures/component/contentview.py

```diff
@@ -43,6 +44,18 @@ def module_ak_cv_lce(module_org, module_lce, module_published_cv):
     return module_ak_with_cv_lce
 
 
+@pytest.fixture(scope='module')
+def module_content_view(module_org, module_repository, module_lce, default_sat):
```

Comment:
as there already is a module_cv fixture would it be better to rename this one to module_cv_repo or something like that to better differentiate the names of those 2 fixtures?

Reply:
I called it `module_cv_repo`, went with logic of `module_ak_cv_lce`